lib/rift/Annex.py

```diff
@@ -280,8 +280,10 @@ def get(self, identifier, destpath):
 
                     if res:
                         with open(tmp_file, 'wb') as f:
-                            for chunk in res.iter_content(chunk_size=8192):
+                            chunk = res.raw.read(8192)
```

Comment:
suggest: maybe add code comment above to explain why we're not using standard iter_content() in this case?